src/routes/_components/status/Media.html

```diff
@@ -141,6 +166,7 @@
     },
     data: () => ({
       oneTransparentPixel: ONE_TRANSPARENT_PIXEL,
+      decodedBlurhash: ONE_TRANSPARENT_PIXEL,
```

Comment:
`ONE_TRANSPARENT_PIXEL` is used while the async computation is pending.

Reply:
Good workaround!